src/components/Buttons/AddToCart.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import { useGlobalContext } from '../../context';
+import { useApi } from '../../hooks/useApi';
+
+import Button from '@mui/material/Button';
+
+const AddToCart = ({ product }) => {
```

Comment:
Probably a little knit-picky, but the action this component is doing is `addToCart` maybe it would be more clear to name the component `AddToCartButton` instead of `AddToCart` -- I'll leave this call up to you.